specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/Diagnostics.json

```diff
@@ -282,6 +282,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/detectorProperties/authConfigsApi/": {
+      "get": {
+        "tags": [
+          "ContainerApps",
+          "Diagnostics"
+        ],
+        "summary": "Get the authentication configurations of a Container App.",
+        "operationId": "ContainerAppsDiagnostics_GetAuthConfigs",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppName",
+            "in": "path",
+            "description": "Name of the Container App.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "./AuthConfigs.json#/definitions/AuthConfig"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Container App's auth configs": {
+            "$ref": "./examples/AuthConfigs_Get.json"
+          }
```

Comment:
so u are adding this API

    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/detectorProperties/authConfigsApi/": {


and pointing it to use this example

./examples/AuthConfigs_Get.json



which exercises this path

        "id": "/subscriptions/651f8027-33e8-4ec4-97b4-f6e9f3dc8744/resourceGroups/workerapps-rg-xj/providers/Microsoft.App/containerApps/testcanadacentral/authconfigs/current",




which is different from

/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/detectorProperties/authConfigsApi



so its not clear to me what u mean by reusing existing examples

Reply:
Thanks. Added the examples for the routes